src/libs/colors.js

```diff
@@ -1,105 +1,34 @@
-export default {
-  lightGreen: [
-    '#b0d239',
-    '#bcd957',
-    '#c8e075',
-    '#d4e793',
-    '#e0edb0',
-    '#ecf4ce',
-    '#f8fbec'
-  ],
-
-  green: [
-    '#5c912e',
-    '#74ae43',
-    '#89bb60',
-    '#9ec77c',
-    '#b3d398',
-    '#c8dfb4',
-    '#ddebd0',
-    '#f2f8ed'
-  ],
-
-  darkGreen: [
-    '#359448',
-    '#54a464',
-    '#72b47f',
-    '#91c59b',
-    '#afd5b6',
-    '#cde5d2',
-    '#ecf5ee'
-  ],
-
-  orange: [
-    '#f7981c',
-    '#f9a83e',
-    '#fab760',
-    '#fbc783',
-    '#fcd6a5',
-    '#fde6c7',
-    '#fff5e9'
-  ],
-
-  red: [
-    '#db3214',
-    '#e15138',
-    '#e6705b',
-    '#ec8f7e',
-    '#f1ada1',
-    '#f6ccc5',
-    '#fcebe8'
-  ],
-
-  grayBlue: [
-    '#c5d0de',
-    '#ced8e3',
-    '#d7dfe8',
-    '#e0e6ed',
-    '#e8edf2',
-    '#f1f4f7',
-    '#fafbfc'
-  ],
-
-  gray: [
-    '#333f52',
-    '#525c6c',
-    '#707986',
-    '#8f96a1',
-    '#aeb3ba',
-    '#cccfd4',
-    '#ebedef'
-  ],
-
-  blue: [
-    '#0086c1',
-    '#2699cb',
-    '#4cabd4',
-    '#73bddd',
-    '#99cfe7',
-    '#bfe1f0',
-    '#e6f4f9'
-  ],
-
-  darkBlue: [
-    '#224f83',
-    '#51719c',
-    '#7d95b5',
-    '#a8b8ce',
-    '#d4dce6',
-    '#eaedf3'
-  ],
-
-  purple: [
-    '#8a6194',
-    '#a180a9',
-    '#b8a0be',
-    '#d1bfd4',
-    '#e8dfea',
-    '#f3eff4'
-  ],
-
-  slate: '#8399a4',
-  brick: '#c35b69',
+import Color from 'color'
+import _ from 'lodash/fp'
+import { getAppName } from 'src/libs/logos'
+
+
+const ALL_COLORS = ['primary', 'secondary', 'accent', 'success', 'warning', 'danger', 'light', 'dark']
+
+const baseColors = {
+  primary: '#4d72aa',
+  secondary: '#6d6e70',
+  accent: '#4d72aa',
+  success: '#74ae43',
+  warning: '#f7981c',
+  danger: '#db3214',
+  light: '#e9ecef',
+  dark: '#333f52'
+}
 
-  gradient: 'linear-gradient(180deg, #74ae43 0%, #359448 100%)'
+const colorPalettes = {
+  Terra: {
```

Comment:
Nit: I feel like Terra should use unaltered base colors with FireCloud overriding primary.